tools/tertiary-analysis/sc3/sc3-sc3-kmeans.xml

```diff
@@ -1,18 +1,18 @@
-<tool id="sc3-sc3-kmeans" name="SC3 kmeans" version="0.1.0">
-    <description>Perform k-Means clustering on cells</description>
+<tool id="sc3-sc3-kmeans" name="SC3 kmeans" version="@TOOL_VERSION@+galaxy0">
+    <description>Perform k-means clustering on single cell datasets</description>
     <macros>
-        <import>macros.xml</import>
+        <import>sc3-macros.xml</import>
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
         sc3-sc3-kmeans.R -i $input1.rds -k "$input2" -o "$output1" 
```

Comment:
Does this command works? What is `$input1.rds`?

Reply:
Indeed, this was not working. I have fixed the syntax for this and all other wrappers as well.